homeassistant/components/media_player/strings.json

```
@@ -29,6 +29,116 @@
         "idle": "[%key:common::state::idle%]",
         "standby": "[%key:common::state::standby%]",
         "buffering": "Buffering"
+      },
+      "state_attributes": {
+        "app_id": {
+          "name": "App ID"
+        },
+        "app_name": {
+          "name": "App name"
+        },
+        "entity_picture_local": {
+          "name": "Local accessible entity picture"
+        },
+        "groups_members": {
+          "name": "Group members"
+        },
+        "is_volume_mutes": {
+          "name": "Muted"
+        },
+        "media_album_artist": {
+          "name": "Album artist"
+        },
+        "media_album_name": {
+          "name": "Album"
+        },
+        "media_artist": {
+          "name": "Artist"
+        },
+        "media_channel": {
+          "name": "Channel"
+        },
+        "media_content_id": {
+          "name": "Content ID"
+        },
+        "media_content_type": {
+          "name": "Content type",
+          "state": {
+            "album": "Album",
+            "app": "App",
+            "artist": "Artist",
+            "channel": "Channel",
+            "channels": "Channels",
+            "composer": "Composer",
+            "contributing_artist": "Contributing artist",
+            "episode": "Episode",
+            "game": "Game",
+            "genre": "Genre",
+            "image": "Image",
+            "movie": "Movie",
+            "music": "Music",
+            "playlist": "Playlist",
+            "podcast": "Podcast",
+            "season": "Season",
+            "track": "Track",
+            "tvshow": "TV show",
+            "url": "URL",
+            "video": "Video"
+          }
+        },
+        "media_duration": {
+          "name": "Duration"
+        },
+        "media_episode": {
+          "name": "Episode"
+        },
+        "media_playlist": {
+          "name": "Playlist"
+        },
+        "media_position": {
+          "name": "Position"
+        },
+        "media_position_updated_at": {
+          "name": "Position updated at"
```

```suggestion
          "name": "Position updated"
```

(note, user should never see this)

---

That is not correct, it can be used in the automation editor (and it will translate it, when used)